torch/hub.py

```diff
@@ -209,6 +212,58 @@ def _get_cache_or_reload(github, force_reload, verbose=True, skip_validation=Fal
 
     return repo_dir
 
+def _get_trusted_input(repo):
```

Comment:
Could we just patch `input` instead? When we can, it's preferable to not alter the code just because of testing constraints.

Reply:
Nope, I get

```python
TypeError: Need a valid target to patch. You supplied: 'input'
```

Reply:
Perhaps we need to specify builtins.input or something like that?